### PR TITLE
Add server profile template table for PhysicalServer

### PR DESCRIPTION
**Pull request by eilam20**

### CODE REVIEW

**Comment by agrare** [comment]
> Merge branch 'ManageIQ:master' into add_server_profile_template

@eilam20 it looks like you did a merge here instead of a rebase can you drop that commit and rebase your branch on master rather than merging master into your branch?

File: spec/migrations/20221117080009_create_physical_server_profile_templates_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    migrate
+
+    # Ensure data exists
+    #
+    # Example:
+    #
+    #   expect(MyModel.count).to eq(0)
+  end
+end
```
[the remark this answers is not in the document]
Comment (tzurvaza) [reply spec/migrations/20221117080009_create_physical_server_profile_templates_spec.rb]: Done 👍

**Comment by agrare** [comment]
Now you have two merge commits @tzurvaza :laughing: I can rebase if you want but you should get used to rebasing on master

**Comment by agrare** [comment]
Looks like you don't have allow maintainer edits enabled so you'll have to fix this yourself

**Comment by agrare** [comment]
Minor but this is `-schema`, the last `fix` commit should probably be rebased into the original commit since it isn't really helpful to the history or showing the progression of work

**Comment by eilam20** [comment]
@agrare I dropped the commits and pushed it again with the rebase.

**Comment by eilam20** [comment]
> Minor but this is `-schema`, the last `fix` commit should probably be rebased into the original commit since it isn't really helpful to the history or showing the progression of work

ok, done

**Review by agrare — APPROVED** [review]

**Pull request merged**